lua/treesitter-context.lua

```diff
@@ -469,8 +460,8 @@ function M.open()
           -- we replace "\n" with " "
           offset = intended_start_row
           -- Add the length of each precending lines
-          for i = 1, intended_start_row do
-            offset = offset + #lines[i] - indents[i]
```

Comment:
Not 100% sure of the context here but why is this removed?

Reply:
line isn't defined in this context. Not sure why this hasn't caused issues, but it is neither a global or a local.

Reply:
Should maybe `lines` -> `context_lines` ?

Reply:
My bad, it should be defined like this:

```diff
diff --git a/lua/treesitter-context.lua b/lua/treesitter-context.lua
index 4c21f10..fec6af8 100644
--- a/lua/treesitter-context.lua
+++ b/lua/treesitter-context.lua
@@ -418,6 +418,7 @@ function M.open(update_gutter_with)
     local current_node = context_nodes[i]
     local range = context_ranges[i]
     local indents = context_indents[i]
+    local lines = context_lines[i]
 
     local start_row = range[1]
     local start_col = range[2]
```

Revert the removal, I'll work on it this afternoon (I promise), I'll merge as soon as that and the typo are done, works fine for the rest.

Reply:
How about the latest version where I've just commented the line and added a FIXME?

Otherwise I'll revert.

Reply:
Yep, good. Thanks

Reply:
Done